client/components/site-selector/index.jsx

```diff
@@ -368,6 +368,12 @@ class SiteSelector extends Component {
 	}
 
 	render() {
+		// Render an empty div.site-selector element as a placeholder. It's useful for lazy
+		// rendering of the selector in sidebar while keeping the on-appear animation work.
+		if ( this.props.isPlaceholder ) {
```

Comment:
We should document this prop by adding it to `propTypes`  and README.

Reply:
I added a proptype. The `SiteSelector`'s README is just one line description of the component, so I didn't add anything there.